packages/@biomejs/js-api/tests/lintContent.test.ts

```diff
@@ -20,4 +20,21 @@ describe("Biome WebAssembly lintContent", () => {
 
 		expect(result.diagnostics).toMatchSnapshot("lint diagnostics");
 	});
+
+	it("should not fix lint issues if fixFileMode is undefined/omitted", () => {
+		const result = biome.lintContent('let a = "foo " + Date.now() + " bar"', {
+			filePath: "example.js",
+		});
+
+		expect(result.content).toMatchSnapshot("original content");
+	});
+
+	it("should fix lint issues and return new content if fileFileMode is SafeAndUnsafeFixes", () => {
+		const result = biome.lintContent('let a = "foo " + Date.now() + " bar"', {
+			filePath: "example.js",
+			fixFileMode: "SafeAndUnsafeFixes",
+		});
+
+		expect(result.content).toMatchSnapshot("fixed content");
+	});
```

Comment:
We should add another test. A content that contains code that triggers a **safe and unsafe** fix, and then attempt to fix only the **safe fixes**.

Then, should assert that:
- we fixed only the safe fix
- there's still a diagnostic about the the rule that triggered the unsafe fix